pkgs/top-level/all-packages.nix

```diff
@@ -7940,6 +7940,8 @@ with pkgs;
 
   mtm = callPackage ../tools/misc/mtm { };
 
+  p2pool = callPackage ../applications/misc/p2pool { };
```

Comment:
If the package is considered an "application", this is the wrong section.
Look for the comment at the top of `all-packages.nix`

Reply:
Looks good!